### ISSUE TITLE
widget to drop a pin on the map and get the coordinate

### ISSUE THREAD

**Issue by EddyIonescu** [trynmaps/tryn-react#33]
needed for https://github.com/trynmaps/orion/wiki/Point-Reliability

**Comment by EddyIonescu** [comment]
Just to simplify things, do you want to add the pin ux first then add the API call in another PR?
Also, the latter requires getting familiar with Relay, which might take some getting used to...

**Comment by matthewpalamos** [comment]
Yeah thats probably a better idea. Did you have any specific location the should pin UX should be put by?

**Comment by EddyIonescu** [comment]
I'm fine with it being near the top of the sidebar - we can move stuff around once it gets crowded. As for location on the map, it should be somewhere along an existing route (while low priority, snapping onto the closet selected route might be a future issue - for now I think anywhere works)
